packages/ui/src/components/sheet/sheet.web.stories.tsx

```diff
@@ -23,14 +23,17 @@ const meta: Meta<typeof Component> = {
 export default meta;
 
 export function Sheet() {
-  const [isShowing, setIsShowing] = useState(false);
+  const triggerRef = useRef<HTMLButtonElement>(null);
```

Comment:
This may be un-wise, but what do you think about `Sheet` exporting a hook like `useSheet` which returns functions around the refs, eg `{ triggerSheet, closeSheet } = useSheet()`. Feels to me like `Sheet` should better encapsulate it's behaviors. Just a thought.

Really loving the animation!